devfile.yaml

```diff
@@ -1,40 +1,49 @@
 apiVersion: 1.0.0
 metadata:
-  name: dashboard-next
+  generateName: che-dashboard-react
 attributes:
   persistVolumes: 'false'
 components:
-  - id: che-incubator/typescript/1.30.2
-    type: chePlugin
-    alias: typescript-plugin
   - mountSources: true
     endpoints:
-      - name: ud-dev-server
-        port: 3000
-        attributes:
+      - attributes:
           public: 'true'
           protocol: http
           path: /
-    memoryLimit: 3Gi
+        name: dashboard-dev-server
+        port: 3000
+    command:
+      - tail
+      - '-f'
+      - /dev/null
+    memoryLimit: 2Gi
     type: dockerimage
-    alias: ud-dev
-    image: 'okurinnyi/che-dashboard-dev:nightly'
+    alias: dash-dev
+    image: 'docker.io/node:10.22.1'
```

Comment:
```suggestion
    image: 'docker.io/node:12'
```

Reply:
OK. I have fixed it.

Reply:
https://github.com/eclipse/che-dashboard/blob/main-react/package.json#L21

Reply:
```
"node": ">=10.20.1"
```